apache/superset
# fix(dashboard): Open "View Chart in Explore" in the same window

### SUMMARY
Fixes: #14672

The function `postForm` introduced in PR #9345 has the target set to `_blank` by default. This PR specifies `_self` instead. There does not seem to be any obvious issue in doing so. Input from @graceguo-supercat is appreciated as a double check.

### TESTING INSTRUCTIONS
1. Open a dashboard
2. Choose a chart and select "View Chart in Explore"
3. Make sure the chart Explore page is opened in the same window

### ADDITIONAL INFORMATION


- [x] Has associated issue: #14672
